src/components/Jokers/CallFriend/CallFriendJoker.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { applyCallFriend } from "store/slices/gamePlaySlice";
+
+import {
+  CLOVER_EMOJI,
+  FRIEND_THINKING,
+  FRIEND_WHISHING_GOOD_LUCK,
+  REPLACE_FROM_AMPERSAND_TO_SEMICOLON_PATTERN,
+} from "utils/constants";
+
+import styles from "./CallFriendJoker.module.css";
+
+const CallFriendJoker = () => {
+  const dispatch = useDispatch();
+  const { correctAnswer, callFriendJoker } = useSelector(
+    (state) => state.gamePlay
+  );
+  const [showSpeechBubble, setShowSpeechBubble] = useState(false);
```

Comment:
Boolean state variables should start with "is" for example "isSpeechBubbleVisible"